tools/frb_internal/lib/src/makefile_dart/generate.dart

```diff
@@ -303,11 +303,17 @@ Future<void> generateRunFrbCodegenCommandGenerate(
     GeneratePackageConfig config) async {
   await _wrapMaybeSetExitIfChanged(config, () async {
     await runPubGetIfNotRunYet(config.package);
+    print("generating with ${config.package}");
+    List<String> features = [];
+    if (config.package == "pure_dart/pure_dart_pde") {
```

Comment:
looks like this is not very correct...

Reply:
I think I see what is happening, I used the same function name for both enabled and disabled feature. Since the rust lib is compiled with enabled feature it returns "test", which passes the test, but in reality the feature is disabled for codegen. Will fix this shortly

Reply:
Actually I dont think thats entirely a correct guess, tests are still passing even after removing that function. I am going to see if I can solve it by calling the code generator during the test itself?

Reply:
The issue here is that requesting a feature in `generateRunFrbCodegenCommandGenerate()` is essentially a no-op because the same feature is already requested from the flutter_rust_bridge.yaml file. 

```yaml
rust_features:
  - internal_feature_for_testing
```
removing this line, along with removing the duplicate `getFeatureTest()` function in the rust code causes tests to fail as expected. I am going to remove the redundant feature handling in generate.dart, etc and move to purely passing features by flutter_rust_bridge.yaml if that is ok